_data/projects.yml

```diff
@@ -58,6 +62,8 @@
   repo: uclaacm/teach-la-website
   description: "This website! Built with love and HTML, CSS/SASS, and Jekyll (a static site generator)."
   lead:
+    - name: Vivian
+      hash: vha
     - name: Matt
```

Comment:
Can remove since they're underneath the previous boards section of the teams page.